terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/java/org/terasoluna/gfw/common/codelist/i18n/SimpleI18nCodeListTest.java

```diff
@@ -21,29 +21,49 @@
 import java.util.Locale;
 import java.util.Map;
 
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Lazy;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(locations = {
         "classpath:org/terasoluna/gfw/common/codelist/i18n/simpleI18nCodeList.xml" })
+@Lazy
 public class SimpleI18nCodeListTest {
     @Autowired
     @Qualifier("CL_testSetRows01")
     protected SimpleI18nCodeList testSetRows01;
 
+    @Autowired
+    @Qualifier("CL_testSetRows02")
+    protected SimpleI18nCodeList testSetRows02;
+
     @Autowired
     @Qualifier("CL_testSetRowsByCodeList01")
     protected SimpleI18nCodeList testSetRowsByCodeList01;
 
+    @Autowired
+    @Qualifier("CL_testSetRowsByCodeList02")
+    protected SimpleI18nCodeList testSetRowsByCodeList02;
+
     @Autowired
     @Qualifier("CL_testSetColumns01")
     protected SimpleI18nCodeList testSetColumns01;
 
+    @Autowired
+    @Qualifier("CL_testSetColumns02")
+    protected SimpleI18nCodeList testSetColumns02;
```

Comment:
- [ ] Why do you need to test three patterns as rows,rowsbyCodeList,columns.
If you want to test locale resolution, it is not related.

Reply:
I deleted it because it became unnecessary.
f771b3a